modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -80,7 +80,12 @@
       "type": "object",
       "properties": {
         "name":  { "$ref": "#/definitions/hlrCreateRepresentativeName" },
-        "phone": { "$ref": "#/definitions/hlrCreatePhone" }
+        "phone": { "$ref": "#/definitions/hlrCreatePhone" },
+        "emailAddressText": {
```

Comment:
I thought we'd changed both of these to `email` to simplify?

Reply:
ha i'm working on that now

Reply:
done